src/envoy/http/service_control/client_cache.h

```diff
@@ -30,6 +30,9 @@ namespace envoy {
 namespace http_filters {
 namespace service_control {
 
+using ::google::api::servicecontrol::v1::CheckResponse;
+using ::google::protobuf::util::Status;
```

Comment:
Per Google style guide,  not allow to use namespace "using" in header files.  It is easy to cause problem.

Reply:
Done